packages/manager/modules/pci/src/components/project/storages/databases/backup.class.js

```diff
@@ -0,0 +1,42 @@
+import includes from 'lodash/includes';
+import 'moment';
+
+export default class Backup {
+  constructor({ createdAt, id, status, description, size }) {
+    Object.assign(this, {
+      createdAt,
+      id,
+      status,
+      description,
+      size,
+    });
+  }
```

Comment:
To avoid two times modification if we have a new attribute and long constructor i think it's better to write it like this.

```suggestion
  constructor(backup) {
    Object.assign(this, {
      createdAt: backup.createdAt,
      id: backup.id,
      status: backup.status,
      description: backup.description,
      size: backup.size,
    });
  }
```

Reply:
@mohammed-zahaf It's done that way so we exactly know what to expect in `backup` as properties